tests/foreman/cli/test_remoteexecution.py

```diff
@@ -180,7 +180,9 @@ def test_positive_run_job_effective_user_by_ip(self, fixture_vmsetup):
         assert username == result.stdout[0]
 
     @pytest.mark.tier3
-    @pytest.mark.parametrize('fixture_vmsetup', [{'nick': 'rhel7'}], indirect=True)
+    @pytest.mark.parametrize(
+        'fixture_vmsetup', [{'nick': 'rhel7'}, {'nick': 'rhel7_fips'}], indirect=True
```

Comment:
I think it's worth providing an `ids` list to parametrize here, right now the parameter ID's don't really help identify the difference between the test nodes. 

`test_positive_run_custom_job_template_by_ip[fixture_vmsetup0]`

This can be as simple or fancy as you want, really.

```
pytest.mark.parametrize('fixture_vmsetup', [{'nick':'rhel7'}, {'nick': 'rhel7_fips'}], ids=['rhel', 'rhel_fips'])
```

```
nick_params = [{'nick':'rhel7'}, {'nick': 'rhel7_fips'}]
pytest.mark.parametrize('fixture_vmsetup', nick_params, ids=[n['nick'] for n in nick_params]
```

Reply:
Not strictly required btw, if somebody else reviews and merges before you add it that's fine. But using parameter IDs is helpful when analyzing test results. Nobody will remember whether FIPS was the 0th or 1st parameter set.